src/main/kotlin/com/openlattice/chronicle/converters/PostgresDownloadCsvHttpMessageConverter.kt

```diff
@@ -105,7 +105,7 @@ class PostgresDownloadCsvHttpMessageConverter : AbstractGenericHttpMessageConver
         val schemaBuilder = CsvSchema.builder()
         val columns: List<String> = iterable.columnAdvice
         columns.forEach(Consumer { col: String ->
-            schemaBuilder.addColumn(col, csvTypes.getValue(resolutionMap.getValue(col)))
+            schemaBuilder.addColumn(col, csvTypes.getValue(resolutionMap.getOrDefault(col, PostgresDatatype.TEXT)))
```

Comment:
Note that this implementation requires a change to PostgresDownloadCsvHttp... If adding a default PostgresDataType will cause an issue in other places where the converter is used, I can tweak TUD downloads to not need this. Although the workaround would not be as concise / flexible.